eventbrite.py

```diff
@@ -0,0 +1,422 @@
+    url = f"{EVENTBRITE_API_BASE_URL}events/{event_id}/ticket_classes/{ticket_class_id}/"
+    headers = {
+        'Authorization': eventbrite_api_token,
+        'Content-Type': 'application/json'
+    }
+    
+    # Make the POST request
+    resp = http.request('POST', url, headers=headers, body=ticket_details)
+    
+    if resp.status != 200:
+        error_data = json.loads(resp.data.decode('utf-8'))
+        error_detail = error_data.get('error_detail', {})
+        return {
+            "error": error_data.get('error', 'Unknown Error'),
+            "error_description": error_data.get('error_description', 'An error occurred'),
+            "error_detail": error_detail,
+            "status_code": resp.status
+        }
+    data = resp.data.decode('utf-8')
+    return json.loads(data)
+
+def quick_create_event(http, eventbrite_api_token, organization_id, title, description, cost_cents=None, publish=True):
+    # Create the event details JSON
+    event_detail = {
+        "event": {
+            "name": {
+                "html": f"<p>{title}</p>"
+            },
+            "description": {
+                "html": f"<p>{description}</p>"
+            },
+            "start": {
+                "timezone": "UTC",
+                "utc": "2023-11-12T02:00:00Z"
+            },
+            "end": {
+                "timezone": "UTC",
+                "utc": "2023-11-30T02:00:00Z"
+            },
+            "currency": "AUD",
+            "online_event": False,
+            "organizer_id": "",
+            "listed": False,
+            "shareable": False,
+            "invite_only": False,
+            "show_remaining": True,
+            "password": "12345",
+            "capacity": 100,
+            "is_reserved_seating": False,
+            "is_series": False,
+            "show_pick_a_seat": False,
+            "show_seatmap_thumbnail": False,
+            "show_colors_in_seatmap_thumbnail": False,
+            "locale": "en_AU"
+        }
+    }
+
+    # Convert the event details to a JSON string
+    event_detail_json = json.dumps(event_detail)
+
+    # Create the event
+    new_event = create_event(http,eventbrite_api_token, organization_id, event_detail_json)
+
+    # Extract the event_id from the response
+    event_id = new_event['id']
+
+    # Create the ticket class details JSON
+    ticket_detail = {
+        "ticket_class": 
+        {
+            "name": "API Test1",
+            "donation": "false",
+            "quantity_total": "100",
+            "minimum_quantity": "1",
+            "maximum_quantity": "10",
+            "hidden": "false",
+            "auto_hide": "false",
+            "auto_hide_before": "",
+            "auto_hide_after": "",
+            "sales_channels": ["online","atd"],
+            "hide_sale_dates": "false",
+            "delivery_methods": ["electronic"],
+            "include_fee": "false",
+            "sales_start": "2023-11-09T08:00:00Z",
```

Comment:
How does the ticket sales time affect the event start time?

Reply:
I believe it's the date when tickets start being available for sale.
sales_start | datetime | Date and time that sales for this Ticket Class begin.